### ISSUE TITLE
TPCircularBuffer is not supported on OSX

### ISSUE THREAD

**Issue by Boggartfly** [syedhali/EZAudio#257]
```
The platform of the target `Pods` (OS X 10.11) is not compatible with `TPCircularBuffer (1.3)`, which does not support `osx`.
```
I'm getting this error when I'm running pod install.

Will this be a problem? Can I continue with the project without worrying that it will mess with my existing source?

**Comment by syedhali** [comment]
Hi @Boggartfly, good catch. I just submitted a PR to the TPCircularBuffer repo with a fix, but for now I've just released a new version of EZAudio (1.1.4) that addresses this issue by using TPCircularBuffer 1.1

**Comment by Boggartfly** [comment]
Yep, no problem man!
